pkgs/development/python-modules/mat2/default.nix

```diff
@@ -0,0 +1,76 @@
+{ lib
+, buildPythonPackage
+, python
+, pythonOlder
+, fetchFromGitLab
+, substituteAll
+, bubblewrap
+, exiftool
+, ffmpeg
+, mime-types
+, wrapGAppsHook
+, gdk-pixbuf
+, gobject-introspection
+, librsvg
+, poppler_gi
+, mutagen
+, pygobject3
+, pycairo
+}:
+
+buildPythonPackage rec {
+  pname = "mat2";
+  version = "0.12.0";
+
+  disabled = pythonOlder "3.5";
+
+  src = fetchFromGitLab {
+    domain = "0xacab.org";
+    owner = "jvoisin";
+    repo = "mat2";
```

Comment:
```suggestion
    repo = pname;
```

Reply:
I don't see the benefit.

Reply:
Both ways are fine but I personally prefer to not reuse pname because they are not really correlated.